packages/react-components/react-avatar/src/components/AvatarGroup/useAvatarGroupStyles.ts

```diff
@@ -18,34 +22,179 @@ const useStyles = makeStyles({
     display: 'inline-flex',
     position: 'relative',
   },
+  pie: {
+    clipPath: 'circle(50%)',
+  },
+});
+
+/**
+ * Styles for overflow button slot.
+ */
+const useOverflowButtonStyles = makeStyles({
+  base: {
+    display: 'inline-flex',
+    position: 'relative',
+    flexShrink: 0,
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: tokens.colorNeutralForeground1,
+    backgroundColor: tokens.colorNeutralBackground1,
+    ...shorthands.borderWidth(`var(${avatarGroupOverflowButtonBorderVar})`),
+    ...shorthands.borderColor(tokens.colorNeutralStroke1),
+    ...shorthands.borderRadius(tokens.borderRadiusCircular),
+    ...shorthands.borderStyle('solid'),
+    ...shorthands.padding(tokens.spacingHorizontalNone),
+  },
+
+  // These styles match the default button styles
+  focusIndicator: createCustomFocusIndicatorStyle({
+    ...shorthands.borderColor('transparent'),
+    outlineColor: 'transparent',
+    outlineWidth: tokens.strokeWidthThick,
+    outlineStyle: 'solid',
+    boxShadow: `${tokens.shadow4} 0 0 0 2px ${tokens.colorStrokeFocus2}`,
+    zIndex: 1,
+  }),
+
+  states: {
+    ':hover': {
+      color: tokens.colorNeutralForeground1Hover,
+      backgroundColor: tokens.colorNeutralBackground1Hover,
+      ...shorthands.borderColor(tokens.colorNeutralStroke1Hover),
+    },
+    ':active': {
+      color: tokens.colorNeutralForeground1Pressed,
+      backgroundColor: tokens.colorNeutralBackground1Pressed,
+      ...shorthands.borderColor(tokens.colorNeutralStroke1Pressed),
+    },
+    ':checked': {
+      color: tokens.colorNeutralForeground1Selected,
+      backgroundColor: tokens.colorNeutralBackground1Selected,
+      ...shorthands.borderColor(tokens.colorNeutralStroke1Selected),
+    },
```

Comment:
Out of curiosity, how does the `:checked` state happen for this button? Does it happen when the popover is open? I don't see where it gets set.

Reply:
I was trying to use it while the popover is open but forgot to remove it since it doesn't work for now.